.github/workflows/release.yml

```diff
@@ -1,7 +1,7 @@
 name: Release to Maven Central
 on:
   release:
-    types: [created]
+    types: [ released ]
```

Comment:
Releasing on `created` event may be a little bit soon, Doing so on release guarantees that the build will take place just after the release is published.